docker-compose.yml

```diff
@@ -96,7 +96,7 @@ services:
 
   fedora-messaging:
     container_name: fedora-messaging
-    image: usercont/packit-service-fedmsg:latest
+    image: docker.io/usercont/packit-service-fedmsg:prod
```

Comment:
Since docker-compose is used for local development, I'd rather see `:stg` here, but this is OK too.

Reply:
Changed.